x-pack/legacy/plugins/apm/public/components/shared/Links/MachineLearningLinks/MLLink.test.tsx

```diff
@@ -26,9 +26,7 @@ jest.spyOn(hooks, 'useCore').mockReturnValue(coreMock);
 
 jest
   .spyOn(savedObjects, 'getAPMIndexPattern')
-  .mockReturnValue(
-    Promise.resolve({ id: 'apm-index-pattern-id' } as savedObjects.ISavedObject)
-  );
+  .mockReturnValue(Promise.resolve({ id: 'apm-index-pattern-id' } as any));
```

Comment:
nit
```suggestion
  .mockResolvedValue({ id: 'apm-index-pattern-id' } as any);
```

Reply:
changed, never knew this was an option :)